lisa/testsuite.py

```diff
@@ -146,7 +147,9 @@ def handle_exception(
                 TestStatus.PASSED,
                 f"{phase}warning: {exception}",
             )
-        elif isinstance(exception, BadEnvironmentStateException):
+        elif isinstance(exception, BadEnvironmentStateException) or isinstance(
+            exception, TcpConnetionException
```

Comment:
[nit] *TcpConne[c]tionException

Reply:
Why separate commit to fix the the typo when the typo was introduced as part of commit which is not yet merged?
Just trying to understand why we need extra commit and not resolve the issue in the commit introducing the problem itself?

Reply:
> Why separate commit to fix the the typo when the typo was introduced as part of commit which is not yet merged? Just trying to understand why we need extra commit and not resolve the issue in the commit introducing the problem itself?

oh nvm, I see there are multiple places in the original code itself that have the typo